client/jetpack-cloud/sections/sidebar-navigation/jetpack-manage.tsx

```diff
@@ -1,61 +1,55 @@
 import { plugins, key, currencyDollar, category } from '@wordpress/icons';
 import { useTranslate } from 'i18n-calypso';
-import page from 'page';
 import NewSidebar from 'calypso/jetpack-cloud/components/sidebar';
-import { itemLinkMatches } from 'calypso/my-sites/sidebar/utils';
 import {
 	JETPACK_MANAGE_DASHBOARD_LINK,
 	JETPACK_MANAGE_PLUGINS_LINK,
 	JETPACK_MANAGE_LICENCES_LINK,
 	JETPACK_MANAGE_BILLING_LINK,
 } from './lib/constants';
+import { redirectPage, isMenuItemSelected } from './lib/sidebar';
+import type { MenuItemProps } from './types';
 
 const JetpackManageSidebar = () => {
 	const translate = useTranslate();
 
 	const onClickMenuItem = ( path: string ) => {
-		page.redirect( path );
+		// TODO: Track event when user clicks on a menu item
```

Comment:
I've made some implementations on tracking events for the sidebar menu. wdyt incorporating a similar technique?

https://github.com/Automattic/wp-calypso/pull/83163/files#diff-f80d6dedf2bba5d1bdedd62dc76125a1616e82e86296498a0375f92cfd2f819bR79

Reply:
Sounds good to me 👍